### PR TITLE
blacklists morphine from dart guns, also haloperidol

### PR DESCRIPTION
**Pull request by kiwedespars**
## About The Pull Request

blacklists morphine from dart guns, also haloperidol

## Why It's Good For The Game

dart guns aren't supposed to be used for combat

## Changelog
:cl:
balance: blacklisted morphine and haloperidol from dart guns
/:cl:

### CODE REVIEW

File: html/changelog.html [hunk 1]
```diff
@@ -516,12 +516,6 @@ <h3 class="author">Putnam3145 updated:</h3>
 			<ul class="changes bgimages16">
 				<li class="bugfix">fixes a major source of lag</li>
 			</ul>
-
-			<h2 class="date">14 July 2021</h2>
-			<h3 class="author">Putnam3145 updated:</h3>
-			<ul class="changes bgimages16">
-				<li class="refactor">Acid is now a component</li>
-			</ul>
```
Comment (BlueWildrose) [inline html/changelog.html]: Don't touch the changelog files, the bot is supposed to do that.

**Review by BlueWildrose — CHANGES_REQUESTED** [review]
I have no opinion on if the dartgun gets nerfed, but leave the changelog HTML be, that's meant for the bot to touch.

File: html/changelog.html (hunk 1 above)
Comment (timothyteakettle) [reply html/changelog.html]: sometimes this just kind of happens randomly on compile

File: code/modules/projectiles/projectile/bullets/dart_syringe.dm [hunk 2]
```diff
@@ -60,6 +60,8 @@
 			if(M.can_inject(null, FALSE, def_zone, piercing)) // Pass the hit zone to see if it can inject by whether it hit the head or the body.
 				..(target, blocked, TRUE)
 				for(var/datum/reagent/medicine/R in reagents.reagent_list) //OD prevention time!
+					if(R.type in GLOB.blacklisted_medchems)
+						return
```
Comment (DeltaFire15) [inline code/modules/projectiles/projectile/bullets/dart_syringe.dm]:
```suggestion
						continue
```

File: code/__DEFINES/med_blacklist.dm [hunk 3]
```diff
@@ -0,0 +1,3 @@
+GLOBAL_LIST_INIT(blacklisted_medchems, list(
+	/datum/reagent/medicine/morphine, /datum/reagent/medicine/haloperidol,	//harmful chemicals in medicine
+	))
```
Comment (DeltaFire15) [inline code/__DEFINES/med_blacklist.dm]: Should probably just be moved to ../__DEFINES/reagents.dm

File: html/changelog.html (hunk 1 above)
Comment (DeltaFire15) [inline html/changelog.html]: Restore the previous here, changelog sometimes gets a bit wacky on local.

**Review by DeltaFire15 — CHANGES_REQUESTED** [review]
PR's pretty agreeable since syringe gun users just swapped to morphine smartdarts, some minor things though.

File: html/changelog.html [hunk 4]
```diff
@@ -516,6 +516,12 @@ <h3 class="author">Putnam3145 updated:</h3>
 			<ul class="changes bgimages16">
 				<li class="bugfix">fixes a major source of lag</li>
 			</ul>
+
+			<h2 class="date">14 July 2021</h2>
+			<h3 class="author">Putnam3145 updated:</h3>
+			<ul class="changes bgimages16">
+				<li class="refactor">Acid is now a component</li>
+			</ul>
```
Comment (zeroisthebiggay) [inline html/changelog.html]:
```suggestion
```
delete this to game

**Review by zeroisthebiggay — CHANGES_REQUESTED** [review]

**Comment by timothyteakettle** [comment]
based pr

**Review by DeltaFire15 — APPROVED** [review]

**Pull request merged**